### PR TITLE
Flake 1.5 experiment

### PR DESCRIPTION
**Pull request by zimbatm**
I'm using this repo to experiment with different ways to organize flake code. Please ignore :)

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
The name clashes with https://github.com/snowflakelinux/snow and I was first a bit confused

**Comment by zimbatm** [comment]
Hey sandro :) thanks, I'll rename it if this goes past the experimentation phase.

File: snow/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+3. No more adding path flakes to the `/nix/store`. This fixes the following
+   problems:
+    1. No more `file not found` because you forgot to `git add` a file.
+    2. No more adding 2GB monorepos to the /nix/store.
+    3. Evaluation errors now point to the right file location.
+   In order to keep the evaluation relatively pure, we propose to instead:
+    1. Sandbox the Nix evaluator so that relative paths cannot reach outside
+       of the flake root directory.
+    2. Introduce a new type of string context for non-store paths, so that
+       they explode when added to a derivation.
```
[the remark this answers is not in the document]
Comment (zimbatm) [reply snow/README.md:57]: One remaining difference is that flakes exports git archives to the /nix/store (IFF the flake is a git repo).

With the new system in place, there are more chances of .gitignored files of influencing the build.

**Comment by zimbatm** [comment]
closing this for now, it was a fun experiment

**Pull request closed without merging**